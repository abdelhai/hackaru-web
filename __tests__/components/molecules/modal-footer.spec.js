import Factory from '@/__tests__/__setups__/factory';
import ModalFooter from '@/components/molecules/modal-footer';

describe('ModalFooter', () => {
  it('render correctly', () => {
    const wrapper = new Factory(ModalFooter, {
      slots: {
        default: '<p>Content</p>'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
