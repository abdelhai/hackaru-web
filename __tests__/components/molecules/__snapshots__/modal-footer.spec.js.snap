// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter render correctly 1`] = `
<footer
  class="modal-footer"
>
  <p>
    Content
  </p>
</footer>
`;
